server/pkg/website/read.go

```diff
@@ -82,6 +158,16 @@ func GetFilesPathList(
 	client *node.Client,
 	websiteAddress string,
 ) ([]string, error) {
+	// Try to get from cache first
+	if files, exists := globalFilePathListCache.get(websiteAddress); exists {
+		// Convert map back to slice
+		result := make([]string, 0, len(files))
+		for file := range files {
+			result = append(result, file)
+		}
+		return result, nil
```

Comment:
you can just to this:

`return maps.Keys(files), nil`

Reply:
since go 1.22 the behavior changed and that does not compile

Reply:
stringe because the `maps` library has been added to sdt in go 1.21.
Did you added ìmport "maps"`

Reply:
```
pkg/website/read.go:165:10: cannot use maps.Keys(files) (value of type iter.Seq[string]) as []string value in return statement
```